addons/calendar/tests/test_calendar.py

```diff
@@ -384,3 +385,60 @@ def test_event_allday_activity_timezone(self):
         })
 
         self.assertEqual(str(activity_id.date_deadline), '2018-10-16')
+
+    def test_event_creation_mail(self):
+        """
+        Check that mail are sent to the attendees on event creation
+        Check that mail are sent to the added attendees on event edit
+        Check that mail are NOT sent to the attendees when detaching a recurring event
+        """
+
+        def _test_one_mail_per_attendee(self, m, partners):
+            # check that every attendee receive a (single) mail for the event
+            for partner in partners:
+                mail = self.env['mail.mail'].search([
+                            ('recipient_ids', 'in', partner.id),
+                            ('subject', 'like', m.name),
+                            ])
+
+                self.assertEqual(len(mail), 1)
```

Comment:
Last nitpickings about formatting: put the ]) on the indent level of the definition, remove the newline.